version.txt

```diff
@@ -1 +1 @@
-0.9.1
+0.9.2
```

Comment:
Based on https://semver.org/#summary we should use `0.10.0` here (and also in the CHANGELOG.md  above).

Reply:
Fixed it